pkg/graph/schema.graphql

```diff
@@ -1587,8 +1585,9 @@ type TRBRequestForm {
   component: String
   needsAssistanceWith: String
   hasSolutionInMind: Boolean
-  proposedSolutionDescription: String
```

Comment:
I think changing the name here makes sense. I noticed `UpdateTRBRequestFormInput` didn't have these changes mirrored. That makes errors in `ApplyChanges`
![image](https://user-images.githubusercontent.com/99294987/196226889-e3ced8c5-53f0-4ce7-a7da-5f3eb8d2a48f.png)